src/core/Akka.Tests/Serialization/SerializeAllMessagesSpec.cs

```diff
@@ -0,0 +1,76 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SerializeAllMessagesSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2023 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Event;
+using Akka.TestKit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Tests.Serialization;
+
+public class SerializeAllMessagesSpec : AkkaSpec
+{
+    public SerializeAllMessagesSpec(ITestOutputHelper output) : base("akka.actor.serialize-messages = on", output)
```

Comment:
Have to enable `akka.actor.serialize-messages=on` in order for this to even be an issue.